thoth/messaging/adviser_trigger.py

```diff
@@ -55,7 +50,7 @@ class MessageContents(BaseMessageContents):  # type: ignore
         github_base_repo_url = attr.ib(type=Optional[str], default=None)
         re_run_adviser_id = attr.ib(type=Optional[str], default=None)
         source_type = attr.ib(type=Optional[str], default=None)
-        version = attr.ib(type=str, default="v1", init=False)
+        version = attr.ib(type=str, default="v2", init=False)
```

Comment:
this requires changes in investigator to be able to read this version I think. cc @KPostOffice

Reply:
https://github.com/thoth-station/investigator/blob/b51e4acacc2f2e2bda80a65527853ac1328a7e3d/thoth/investigator/adviser_trigger/investigate_adviser_trigger.py#L37

Reply:
Anything else besides https://github.com/thoth-station/investigator/pull/445/ is needed?

Reply:
Adjusted in https://github.com/thoth-station/investigator/pull/445/

Reply:
> Anything else besides [thoth-station/investigator#445](https://github.com/thoth-station/investigator/pull/445) is needed?

Not that comes to my mind :)